BridgeAppSDK/SBAActivityTableViewCell.swift

```diff
@@ -35,32 +35,10 @@ import UIKit
 
 open class SBAActivityTableViewCell: UITableViewCell {
     
-    @IBOutlet weak var uncheckedView: UIView!
-    @IBOutlet weak var checkmarkImageView: UIImageView!
-    @IBOutlet open weak var titleLabel: UILabel!
-    @IBOutlet open weak var subtitleLabel: UILabel!
-    @IBOutlet open weak var timeLabel: UILabel?
+    @IBOutlet open var titleLabel: UILabel!
+    @IBOutlet open var subtitleLabel: UILabel!
+    @IBOutlet open var timeLabel: UILabel?
```

Comment:
shouldn't IBOutlets always be weak? or is Swift smart enough to know that by implication?